src/modules/apps.js

```diff
@@ -224,22 +246,31 @@ export default {
   uninstall: {
     requiredArgs: 'app',
     description: 'Uninstall the specified app',
-    handler: (app) => {
+    options: [
+      {
+        short: 'y',
```

Comment:
There's a lot of stuff that I do repeatedly when using inquirer prompts, we really should wrap them.